src/handlers/ground-truths/find-ground-truths.ts

```diff
@@ -0,0 +1,31 @@
+import { Context } from "../../types";
+import { CodeReviewAppParams, CodeReviewGroundTruthSystemMessage } from "../../types/llm";
+import { CODE_REVIEW_GROUND_TRUTHS_SYSTEM_MESSAGE } from "./prompts";
+import { codeReviewPayloadTypeguard } from "../../types/typeguards";
+import { validateGroundTruths } from "./validate";
+import { logger } from "../../helpers/errors";
+import { createGroundTruthSysMsg } from "./create-system-message";
+
+export async function findGroundTruths(context: Context, params: CodeReviewAppParams): Promise<string[]> {
+  const systemMsgObj = CODE_REVIEW_GROUND_TRUTHS_SYSTEM_MESSAGE;
+
+  // params are deconstructed to show quickly what's being passed to the function
+  if (codeReviewPayloadTypeguard(params)) {
```

Comment:
`params` is already of type `CodeReviewAppParams` so I don't understand why a typeguard is necessary here